README.md

```diff
@@ -313,37 +313,43 @@ Deploy [web validate](https://github.com/retaildevcrews/webvalidate) to drive co
 
 ```bash
 
+# Resource Group Name
+export He_WebV_RG=${He_Name}-rg-webv
+
+# create a new resource group
+az group create -n $He_WebV_RG -l $He_Location
+
 # Add Log Analytics extension
 az extension add -n log-analytics
 
 # create Log Analytics for the webv clients
-az monitor log-analytics workspace create -g $He_App_RG -l $He_Location -n $He_Name -o table
+az monitor log-analytics workspace create -g $He_WebV_RG -l $He_Location -n $He_Name -o table
 
 # retrieve the Log Analytics values using eval $He_LogAnalytics_*
-export He_LogAnalytics_Id='az monitor log-analytics workspace show -g $He_App_RG -n $He_Name --query customerId -o tsv'
-export He_LogAnalytics_Key='az monitor log-analytics workspace get-shared-keys -g $He_App_RG -n $He_Name --query primarySharedKey -o tsv'
+export He_LogAnalytics_Id='az monitor log-analytics workspace show -g $He_WebV_RG -n $He_Name --query customerId -o tsv'
+export He_LogAnalytics_Key='az monitor log-analytics workspace get-shared-keys -g $He_WebV_RG -n $He_Name --query primarySharedKey -o tsv'
 
 # save the environment variables
 ./saveenv.sh -y
```

Comment:
I would move the two exports and ./saveenv up to 318 so that the exports / save are grouped together.

Reply:
That should, but is it alright to have variables 'created' when the parent Log Analytics instance hasn't been created yet?